doc/src/site/sphinx/about.rst

```diff
@@ -6,4 +6,6 @@ Sparkling Water allows users to combine the fast, scalable machine learning algo
 Spark is an elegant and powerful general-purpose, open-source, in-memory platform with tremendous momentum. H2O is an in-memory application for machine learning that is reshaping how people apply math and predictive analytics to their business problems.
 
 Integrating these two open-source environments provides a seamless experience for users who want to make a query  using Spark SQL, feed the results into H2O to build a model and make predictions, and then use the results again in
-Spark. For any given problem, better interoperability between tools provides a better experience.
\ No newline at end of file
+Spark. For any given problem, better interoperability between tools provides a better experience.
+
+Sparkling Water is licensed under the Apache License, Version 2.0. For more information, see https://github.com/h2oai/sparkling-water/blob/master/LICENSE.
```

Comment:
WDYT about adding a link to http://www.apache.org/licenses/LICENSE-2.0?

E.g.: Sparkling Water is licensed under the `Apache License, Version 2.0 <http://www.apache.org/licenses/LICENSE-2.0>`_.

Reply:
Sounds good to me.